### DISCUSSION TITLE
Employer ID not recognized

### DISCUSSION THREAD

**Discussion by zaccharieramzi** [community/community#48561] [GitHub Education]
I would like to apply for an academic discount coupon, since I am a postdoc researcher.
However, everytime I try to upload a new photo of my employer ID the prescreening system fails.
It says that :
```
- The image you used appears to be edited. Manipulated images may not be used. Please try a different image.

- The image you selected does not appear to contain your school name. You may include multiple documents in your image, so long as they are legible. For example, if your original document is not in English then you might take a photo of the original next to an English translation.
```
I have tried several times with different photos and different cameras, always failing : what can I do to bypass the automated screening system?

**Comment by samus-aran** [comment]
Hi @zaccharieramzi 👋 

Thank you for posting your rejection reasons.

Do you have additional evidence that you can show along side your current evidence? As you can imagine there are many types of academic verification and our system is constantly evolving to accommodate this without setting the bar too high. However sometimes we may require a little more evidence from the student applying and we thank you for being patient.

**Reply by zaccharieramzi** [reply]
as I was saying I am not a student but a postdoc researcher, would an employment contract be a good additional evidence?

**Reply by samus-aran** [reply]
Yes but your Academic ID is preferable in terms of evidence. The order of options indicates the preference of evidence with Academic Email being at the top usually followed by Student/Faculty ID. We also ask applicants to ensure that the evidence matches the selected school you are applying with. Does the school's name correspond - consider it's name in English if it is not in English. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.
If you need to translate we accept pre-translated evidence - take a photo of original evidence along side an English translation.

Hope this helps. Let us know how you get on 👍

**Reply by zaccharieramzi** [reply]
I think the main reason is that there is a mismatch between the name of the school recognized from my email address and the actual school I attend. When I try selecting the correct school, it tells me that I am not using the correct email address, while it actually is.

![Screenshot from 2023-03-03 11-52-15](https://user-images.githubusercontent.com/6387497/222702620-43caffd5-2478-43ea-af16-bb4980fab030.png)

My school is not the normal school superior  of Paris-Saclay, but the normal school superior of Paris (i.e. Ecole Normale Superieure).
(not Saclay).

Should I still say normal school superior  of Paris-Saclay and add a document in the photo explaining the confusion?

**Reply by zaccharieramzi** [reply]
Hi @samus-aran ,

Do you think there is a solution to this problem?
I might lose my GitHub copilot access soon and I'd rather not, it's been so helpful for my coding.

Cheers

**Reply by samus-aran** [reply]
Hi @zaccharieramzi, thank you for your patience. 🙂 
We'll pass this possible mismatch of domains onto the GitHub Education team and we'll post an update when available.

**Reply by samus-aran** [reply]
Hi @zaccharieramzi 

We have updated the applied schools' details. We would like to ask you to re-apply and let us know how you get on 👍

**Reply by zaccharieramzi** [reply]
Hi @samus-aran ,

All working now, despite the school still being tagged as "the normal school superior of Paris-Saclay". Thanks for your work!
